src/converters/app-converter.ts

```diff
@@ -35,30 +35,50 @@ export class AppConverter {
     }
 
     convert() {
+        const assetStackIds = this.host.dependencies.flatMap((dep) => dep.name);
+        const stackManifests: StackManifest[] = [];
         for (const stackManifest of this.manifestReader.stackManifests) {
+            // Don't process artifact manifests
```

Comment:
Because we handle the assets as part of the synthesizer. The way it worked before is the synthesizer would "register" the asset by writing it to the artifact manifest. We would then process that here to create the pulumi resources (bucketobject, etc). Now that we create those resources directly as part of the synthesizer we don't want to process them again.

I thought about also no synthesizing the asset manifest at all, but decided to keep it for debugging purposes.

Reply:
Thanks for explaining!